dashboard/app/helpers/levels_helper.rb

```diff
@@ -68,6 +70,10 @@ def get_channel_for(level, user = nil)
     channel_token.try :channel
   end
 
+  def safe_get_channel_for(level, user)
+    get_channel_for(level, user, false)
+  end
```

Comment:
As a quick fix for our production issues this seems fine. I have some additional thoughts on the more long term fix.